common/dockerd.json

```diff
@@ -0,0 +1,11 @@
+{
+    "image": "docker.io/seemethere/engine-community:0.0.0-20180814124044-678d4b3a6d.x86_64",
```

Comment:
This probably shouldn't be pointing to `seemethere`'s hub org.

Reply:
This is currently just for development purposes. I'll have a follow up PR after I finish up Deb packaging as well to make this configurable.